api/handlers/userhandler.py

```diff
@@ -100,6 +105,19 @@ def post(self):
     def _init_storage(self):
         self.storage = containerstorage.ContainerStorage('users', use_object_id=False)
 
+    def _cleanup_user_permissions(self, uid):
```

Comment:
This is making db calls, so I'd like it if it weren't in a handler class. It also would only ever be called by userhandler, so it doesn't really work when added to one of the storage classes. 

A proposal would be to maybe have a `user.py` class under `dao` with static methods that pertain to user objects that require db access.